Library/Homebrew/rubocops/options_cop.rb

```diff
@@ -0,0 +1,20 @@
+require_relative "./extend/formula_cop"
+
+module RuboCop
+  module Cop
+    module FormulaAudit
+      # This cop audits `options` in Formulae
+      class Options < FormulaCop
+        DEPRICATION_MSG = "macOS has been 64-bit only since 10.6"\
```

Comment:
`DEPRECATION_MSG`. I'd probably avoid wrapping this line at 80 characters because it hards readability in this case.